Tests/SentryTests/SentryHubTests.swift

```diff
@@ -370,28 +370,68 @@ class SentryHubTests: XCTestCase {
         XCTAssertEqual(span.sampled, .no)
     }
     
+    @available(*, deprecated) // The test is marked as deprecated to silence the deprecation warning of the initializer
```

Comment:
`l`: Maybe a bit cleaner.
```suggestion
    @available(*, deprecated, message: "The test is marked as deprecated to silence the deprecation warning of the initializer")
```